icinga2/resource_icinga2_host_test.go

```diff
@@ -16,6 +16,7 @@ func TestAccCreateBasicHost(t *testing.T) {
 		hostname      = "terraform-host-1"
 		address       = "10.10.10.1"
 		check_command = "hostalive"
+		groups = ["linux-servers"]
```

Comment:
Per making groups optional I would remove groups from all the tests, and have one dedicated to testing creating a host with a group. I would also create a test to verify it will change/update the group.